.github/workflows/test-from-pypi.yml

```
@@ -6,112 +6,83 @@ on:
     # Run at 03:27 UTC on the 8th and 22nd of every month
     - cron: '27 3 8,22 * *'
 
-env:
-  # Temporary workaround prior to release of Traits 8.0
-  # xref: enthought/traits#1742
-  ETS_QT4_IMPORTS: 1
-
 jobs:
   test-pypi-sdist:
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
         python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
-        python-architecture: [x86, x64]
+        os: [ubuntu-latest, windows-latest, macos-latest, macos-13]
+        python-architecture: [x86, x64, arm64]
```

Potentially this multi-dimensional matrix might work / look a bit cleaner:
```suggestion
        system:
          - os: ubuntu-latest
            architecture: x86
          - os: windows-latest
            architecture: x86
          - os: windows-latest
            architecture: x64
          - os: macos-13
            architecture: x86
```
I don't trust GH syntaxt to unpack arrays beyond the first nested set, so I explicitly declared the `windows-latest` architecture options

---

Thank you - adopted exactly this solution after some internal discussion (and before seeing this comment - sorry).